recipes/aws-sdk-cpp/all/patches/0001-disable-sort-links.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/cmake/sdks.cmake b/cmake/sdks.cmake
+index 975281faaa..ebedbcc6e5 100644
+--- a/cmake/sdks.cmake
++++ b/cmake/sdks.cmake
+@@ -236,7 +236,7 @@ function(add_sdks)
+ 
+     # the catch-all config needs to list all the targets in a dependency-sorted order
+     include(dependencies)
+-    sort_links(EXPORTS)
++    # sort_links(EXPORTS)
```

Comment:
What does it fix?

Reply:
Without this patch, I have the following error:

```
$ conan create . aws-sdk-cpp/1.8.130@
...
-- Conan: Using autogenerated Findxorg.cmake
-- Found xorg: system (found version "system")
-- Dependency OpenSSL already found
-- Pulse audio header files have been detected, included pulse audio as a possible sound driver implementation.
CMake Error at source_subfolder/cmake/dependencies.cmake:15 (get_property):
  INTERFACE_LIBRARY targets may only have whitelisted properties.  The
  property "LINK_LIBRARIES_ALL" is not allowed.
Call Stack (most recent call first):
  source_subfolder/cmake/dependencies.cmake:33 (compute_links)
  source_subfolder/cmake/dependencies.cmake:59 (compute_links)
  source_subfolder/cmake/sdks.cmake:239 (sort_links)
  source_subfolder/CMakeLists.txt:316 (add_sdks)


CMake Error at source_subfolder/cmake/dependencies.cmake:20 (set_property):
  INTERFACE_LIBRARY targets may only have whitelisted properties.  The
  property "LINK_LIBRARIES_ALL" is not allowed.
Call Stack (most recent call first):
  source_subfolder/cmake/dependencies.cmake:33 (compute_links)
  source_subfolder/cmake/dependencies.cmake:59 (compute_links)
  source_subfolder/cmake/sdks.cmake:239 (sort_links)
  source_subfolder/CMakeLists.txt:316 (add_sdks)


CMake Error at source_subfolder/cmake/dependencies.cmake:24 (get_target_property):
  INTERFACE_LIBRARY targets may only have whitelisted properties.  The
  property "LINK_LIBRARIES" is not allowed.
Call Stack (most recent call first):
  source_subfolder/cmake/dependencies.cmake:33 (compute_links)
  source_subfolder/cmake/dependencies.cmake:59 (compute_links)
  source_subfolder/cmake/sdks.cmake:239 (sort_links)
  source_subfolder/CMakeLists.txt:316 (add_sdks)


CMake Error at source_subfolder/cmake/dependencies.cmake:34 (get_target_property):
  INTERFACE_LIBRARY targets may only have whitelisted properties.  The
  property "LINK_LIBRARIES_ALL" is not allowed.
Call Stack (most recent call first):
  source_subfolder/cmake/dependencies.cmake:59 (compute_links)
  source_subfolder/cmake/sdks.cmake:239 (sort_links)
  source_subfolder/CMakeLists.txt:316 (add_sdks)


CMake Error at source_subfolder/cmake/dependencies.cmake:15 (get_property):
  INTERFACE_LIBRARY targets may only have whitelisted properties.  The
  property "LINK_LIBRARIES_ALL" is not allowed.
Call Stack (most recent call first):
  source_subfolder/cmake/dependencies.cmake:33 (compute_links)
  source_subfolder/cmake/dependencies.cmake:59 (compute_links)
  source_subfolder/cmake/sdks.cmake:239 (sort_links)
  source_subfolder/CMakeLists.txt:316 (add_sdks)


CMake Error at source_subfolder/cmake/dependencies.cmake:20 (set_property):
  INTERFACE_LIBRARY targets may only have whitelisted properties.  The
  property "LINK_LIBRARIES_ALL" is not allowed.
Call Stack (most recent call first):
  source_subfolder/cmake/dependencies.cmake:33 (compute_links)
  source_subfolder/cmake/dependencies.cmake:59 (compute_links)
  source_subfolder/cmake/sdks.cmake:239 (sort_links)
  source_subfolder/CMakeLists.txt:316 (add_sdks)


CMake Error at source_subfolder/cmake/dependencies.cmake:24 (get_target_property):
  INTERFACE_LIBRARY targets may only have whitelisted properties.  The
  property "LINK_LIBRARIES" is not allowed.
Call Stack (most recent call first):
  source_subfolder/cmake/dependencies.cmake:33 (compute_links)
  source_subfolder/cmake/dependencies.cmake:59 (compute_links)
  source_subfolder/cmake/sdks.cmake:239 (sort_links)
  source_subfolder/CMakeLists.txt:316 (add_sdks)


CMake Error at source_subfolder/cmake/dependencies.cmake:34 (get_target_property):
  INTERFACE_LIBRARY targets may only have whitelisted properties.  The
  property "LINK_LIBRARIES_ALL" is not allowed.
Call Stack (most recent call first):
  source_subfolder/cmake/dependencies.cmake:59 (compute_links)
  source_subfolder/cmake/sdks.cmake:239 (sort_links)
  source_subfolder/CMakeLists.txt:316 (add_sdks)


CMake Error at source_subfolder/cmake/dependencies.cmake:15 (get_property):
  INTERFACE_LIBRARY targets may only have whitelisted properties.  The
  property "LINK_LIBRARIES_ALL" is not allowed.
Call Stack (most recent call first):
  source_subfolder/cmake/dependencies.cmake:33 (compute_links)
  source_subfolder/cmake/dependencies.cmake:59 (compute_links)
  source_subfolder/cmake/sdks.cmake:239 (sort_links)
  source_subfolder/CMakeLists.txt:316 (add_sdks)


CMake Error at CMakeLists.txt:7 (add_subdirectory):
  INTERFACE_LIBRARY targets may only have whitelisted properties.  The
  property "LINK_LIBRARIES_ALL" is not allowed.


CMake Error at source_subfolder/cmake/dependencies.cmake:24 (get_target_property):
  INTERFACE_LIBRARY targets may only have whitelisted properties.  The
  property "LINK_LIBRARIES" is not allowed.
Call Stack (most recent call first):
  source_subfolder/cmake/dependencies.cmake:33 (compute_links)
  source_subfolder/cmake/dependencies.cmake:59 (compute_links)
  source_subfolder/cmake/sdks.cmake:239 (sort_links)
  source_subfolder/CMakeLists.txt:316 (add_sdks)


CMake Error at source_subfolder/cmake/dependencies.cmake:34 (get_target_property):
  INTERFACE_LIBRARY targets may only have whitelisted properties.  The
  property "LINK_LIBRARIES_ALL" is not allowed.
Call Stack (most recent call first):
  source_subfolder/cmake/dependencies.cmake:59 (compute_links)
  source_subfolder/cmake/sdks.cmake:239 (sort_links)
  source_subfolder/CMakeLists.txt:316 (add_sdks)


-- Configuring incomplete, errors occurred!
See also "/home/conan/.conan/data/aws-sdk-cpp/1.8.130/_/_/build/d256d3ddc9c58ec327d60900f44081371ed80e6e/CMakeFiles/CMakeOutput.log".
See also "/home/conan/.conan/data/aws-sdk-cpp/1.8.130/_/_/build/d256d3ddc9c58ec327d60900f44081371ed80e6e/CMakeFiles/CMakeError.log".
aws-sdk-cpp/1.8.130:
aws-sdk-cpp/1.8.130: ERROR: Package 'd256d3ddc9c58ec327d60900f44081371ed80e6e' build failed
aws-sdk-cpp/1.8.130: WARN: Build folder /home/conan/.conan/data/aws-sdk-cpp/1.8.130/_/_/build/d256d3ddc9c58ec327d60900f44081371ed80e6e
ERROR: aws-sdk-cpp/1.8.130: Error in build() method, line 373
	cmake = self._configure_cmake()
while calling '_configure_cmake', line 367
	self._cmake.configure()
	ConanException: Error 1 while executing cd '/home/conan/.conan/data/aws-sdk-cpp/1.8.130/_/_/build/d256d3ddc9c58ec327d60900f44081371ed80e6e' && cmake -G "Unix Makefiles" -DCMAKE_BUILD_TYPE="Release" -DCONAN_IN_LOCAL_CACHE="ON" -DCONAN_COMPILER="gcc" -DCONAN_COMPILER_VERSION="8" -DCONAN_CXX_FLAGS="-m64" -DCONAN_SHARED_LINKER_FLAGS="-m64" -DCONAN_C_FLAGS="-m64" -DCONAN_LIBCXX="libstdc++" -DBUILD_SHARED_LIBS="OFF" -DCMAKE_INSTALL_PREFIX="/home/conan/.conan/data/aws-sdk-cpp/1.8.130/_/_/package/d256d3ddc9c58ec327d60900f44081371ed80e6e" -DCMAKE_INSTALL_BINDIR="bin" -DCMAKE_INSTALL_SBINDIR="bin" -DCMAKE_INSTALL_LIBEXECDIR="bin" -DCMAKE_INSTALL_LIBDIR="lib" -DCMAKE_INSTALL_INCLUDEDIR="include" -DCMAKE_INSTALL_OLDINCLUDEDIR="include" -DCMAKE_INSTALL_DATAROOTDIR="share" -DCONAN_CMAKE_POSITION_INDEPENDENT_CODE="ON" -DCMAKE_MODULE_PATH="/home/conan/.conan/data/aws-sdk-cpp/1.8.130/_/_/build/d256d3ddc9c58ec327d60900f44081371ed80e6e" -DCMAKE_EXPORT_NO_PACKAGE_REGISTRY="ON" -DCONAN_EXPORTED="1" -DBUILD_ONLY="core;access-management;identity-management;queues;s3-encryption;text-to-speech;transfer" -DBUILD_DEPS="False" -DENABLE_UNITY_BUILD="True" -DENABLE_TESTING="False" -DAUTORUN_UNIT_TESTS="False" -DMINIMIZE_SIZE="False" -Wno-dev '/home/conan/.conan/data/aws-sdk-cpp/1.8.130/_/_/build/d256d3ddc9c58ec327d60900f44081371ed80e6e'
```